packages/ui/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["index.ts", "src", "types"],
+  "include": ["index.ts", "src", "../../global.d.ts"],
```

Comment:
If the `global.d.ts` file is only used for `*.css`, wouldn't make it sense to have it in `packages/ui` and have it be called `uiGlobals.d.ts` or something. Doesn't make sense for that file to be at the root dir if it's only used in one package

Reply:
Also needed in react package, it errored during builds because `.css` was not recognised for some imported bits there